OWNERS

```diff
@@ -1,8 +1,5 @@
 approvers:
-  - giuseppe
-  - haircommander
-  - harche
-  - mrunalp
```

Comment:
If we are going to keep @mrunalp as the author of `Cargo.toml`, I think we need to keep him in OWNERS as well.

Reply:
@utam0k The idea was to list only the current active maintainers in the owners file but I left him in as author as he has contributed significantly in the past and I wanted to preserve that. I did not think a lot about this though, so I am happy to do any change you suggest.